### PR TITLE
AudacityTeam.Audacity version 3.2.5.0

### PR DESCRIPTION
**Pull request by Nickster888**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] This PR only modifies one (1) manifest
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

 ###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/103277)

### CODE REVIEW

File: manifests/a/AudacityTeam/Audacity/3.2.5.0/AudacityTeam.Audacity.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: AudacityTeam.Audacity
```
Comment (stephengillie) [inline manifests/a/AudacityTeam/Audacity/3.2.5.0/AudacityTeam.Audacity.installer.yaml:4]:
```suggestion
PackageIdentifier: Audacity.Audacity
```

File: manifests/a/AudacityTeam/Audacity/3.2.5.0/AudacityTeam.Audacity.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: AudacityTeam.Audacity
```
Comment (stephengillie) [inline manifests/a/AudacityTeam/Audacity/3.2.5.0/AudacityTeam.Audacity.locale.en-US.yaml:4]:
```suggestion
PackageIdentifier: Audacity.Audacity
```

File: manifests/a/AudacityTeam/Audacity/3.2.5.0/AudacityTeam.Audacity.yaml [hunk 3]
```diff
@@ -0,0 +1,8 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.4.0.schema.json
+
+PackageIdentifier: AudacityTeam.Audacity
```
Comment (stephengillie) [inline manifests/a/AudacityTeam/Audacity/3.2.5.0/AudacityTeam.Audacity.yaml:4]:
```suggestion
PackageIdentifier: Audacity.Audacity
```

Comment (stephengillie) [reply manifests/a/AudacityTeam/Audacity/3.2.5.0/AudacityTeam.Audacity.yaml:4]: Please update file paths to match this change.

File: manifests/a/AudacityTeam/Audacity/3.2.5.0/AudacityTeam.Audacity.locale.en-US.yaml (hunk 2 above)
Comment (stephengillie) [reply manifests/a/AudacityTeam/Audacity/3.2.5.0/AudacityTeam.Audacity.locale.en-US.yaml:4]: Please update file paths to match this change.

File: manifests/a/AudacityTeam/Audacity/3.2.5.0/AudacityTeam.Audacity.installer.yaml (hunk 1 above)
Comment (stephengillie) [reply manifests/a/AudacityTeam/Audacity/3.2.5.0/AudacityTeam.Audacity.installer.yaml:4]: Please update file paths to match this change.

File: manifests/a/AudacityTeam/Audacity/3.2.5.0/AudacityTeam.Audacity.locale.en-US.yaml [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: AudacityTeam.Audacity
+PackageVersion: 3.2.5.0
+PackageLocale: en-US
+Publisher: Audacity Team
+PackageName: Audacity
+License: GNU General Public License
+Copyright: Copyright © 2018. All rights reserved.
+ShortDescription: Audacity 3.2.5
```
Comment (Trenly) [inline manifests/a/AudacityTeam/Audacity/3.2.5.0/AudacityTeam.Audacity.locale.en-US.yaml:11]: Please use the metadata already existing from Audacity.Audacity too

**Comment by ItzLevvie** [comment]
[Policy] Needs-CLA

**Comment by Nickster888** [comment]
Stop contacting me


Sent from Yahoo Mail for iPhone


On Saturday, June 24, 2023, 3:55 PM, microsoft-github-policy-service[bot] ***@***.***> wrote:




Hello @Nickster888,

This PR cannot be merged until you sign the Contributor License Agreement (CLA). More information on this process can be found on the Microsoft Open Source website.

Template: msftbot/needsCLA

**Pull request closed without merging**